Xamarin.Forms.ControlGallery.iOS/Properties/AssemblyInfo.cs

```diff
@@ -40,4 +40,5 @@
 [assembly: Xamarin.Forms.ResolutionGroupName (Xamarin.Forms.Controls.Issues.Effects.ResolutionGroupName)]
 
 // Deliberately broken image source and handler so we can test handling of image loading errors
-[assembly: ExportImageSourceHandler(typeof(FailImageSource), typeof(BrokenImageSourceHandler))]
\ No newline at end of file
+[assembly: ExportImageSourceHandler(typeof(FailImageSource), typeof(BrokenImageSourceHandler))]
+[assembly: ExportRenderer(typeof(WkWebView), typeof(Xamarin.Forms.Platform.iOS.WkWebViewRenderer))]
```

Comment:
If we're planning to support the old WebViewRenderer still, we should `#if` this so we can run tests with the old renderer, too.

Reply:
Actually, this is my happy solution to that problem. `WkWebView` is only used in the 1666 unit test. All the existing tests still use `WebView` which uses the old renderer. 

I added `WkWebView` to the test in gallery. It derives from the public `WebView`. So only 1666 will use the `WkWebViewRenderer` during iOS test runs. Everything else should be the same.  

That works, no?